modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -3260,6 +3283,74 @@ void ONNXImporter::parseQMatMul(LayerParams& layerParams, const opencv_onnx::Nod
     addLayer(layerParams, node_proto);
 }
 
+// A * B + C = Y, we require that the dimension of A is [m, k], and the dimension of B is [n, k].
+// And the dim of output Y is [m, n]
+void ONNXImporter::parseQGemm(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto)
+{
+    int ninputs = node_proto.input_size();
+    CV_Assert(ninputs == 8 || ninputs == 9);
```

Comment:
QGemm has 6-9 inputs, but I think this relies on the fact that C is not provided.

Reply:
Because we can't get the corresponding attributes of the input data. Currently, I can only support 8 or 9 inputs.